.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,30 @@
+name: Continuous Integration 
+on:
+  push:
+    paths-ignore:
+      - README.md
+      - ansible/*
+      - .vscode/*
+      - .venv/*
+      - .pytest_cache/*
+      - '**__pycache__/*'
+      - tests/.pytest_cache/*
```

Comment:
These paths are in your `.gitignore` I believe so GitHub should have no knowledge of them, and so while this is no harm it doesn't achieve anything!
```suggestion
```